youtube_dl/extractor/youku.py

```diff
@@ -1,123 +1,215 @@
 # coding: utf-8
-
 from __future__ import unicode_literals
 
-import math
-import random
+import sys
+pyvs = sys.version_info[0]
```

Comment:
It's better to put all Python 2/3 compatibility codes to [compat.py](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/compat.py)